mmv1/third_party/README.md

```diff
@@ -0,0 +1,42 @@
+# Handwritten
+
+## Overview
+
+The Google providers for Terraform have a large number of handwritten go files, written before Magic Modules was used with them. While conversion is ongoing, many resources are still managed by hand.
```

Comment:
```suggestion
The Google providers for Terraform have a large number of handwritten go files, primarily for resources written before Magic Modules was used with them. Most handwritten files are expected to stay handwritten indefinitely, although conversion to a generator may be possible for a limited subset of them.
```

We haven't actively done conversions in a while, so let's message slightly more strongly against doing so.